.config/mkdocs.yml

```diff
@@ -101,6 +101,7 @@ nav:
           - common-practices-tools/software-and-support/trello.md
           - common-practices-tools/software-and-support/zoom.md
       - common-practices-tools/telephone.md
+      - company-policies/sustainability.md
```

Comment:
@mgifford the new page is in the company-policies path but the menu is under 'Our practices and tools'. Which section should it be in?

Reply:
I am will update this.